### DISCUSSION TITLE
Discussions and discussions

### DISCUSSION THREAD

**Discussion by adingbatponder** [meshtastic/Discussions#4] [Off Topic]
What is the difference between 
https://github.com/meshtastic/meshtastic/discussions
![Screenshot from 2024-10-28 22-27-42](https://github.com/user-attachments/assets/d5c0319b-7985-4d68-b1db-00267ff1abd0)

and 
https://github.com/orgs/meshtastic/discussions

![Screenshot from 2024-10-28 22-29-10](https://github.com/user-attachments/assets/4a1af58d-ddac-4361-b429-05f82fc860ff)

?

**Comment by ikearat** [comment]
I was about to wonder this myself. Considering the link on the Discourse page was "broken" for awhile, I am chalking these questions up to "let the dust settle" and will start to care in a few days.

**Comment by rcarteraz** [comment]
> What is the difference between 
> 
> and https://github.com/orgs/meshtastic/discussions
> 

> 
> ?

The first one is the docs repo and that one is specifically used for the blog post comments. I'm looking to see if they can be migrated but for now they should not be used.

It's pretty easy to get to the correct one because it's linked on the main GitHub page for Meshtastic: https://github.com//meshtastic/
<img width="1364" alt="Screenshot 2024-10-28 at 4 25 05 PM" src="https://github.com/user-attachments/assets/7c57c759-e9d7-4826-b336-dd30e78cdbf3">

**Reply by rcarteraz** [reply]
For now, I've removed all other categories and added an announcement banner to the docs repo to direct people here in case they got lost on the way.

**Reply by rcarteraz** [reply]
Okay, blog post comments have been migrated and discussions at meshtastic/meshtastic have been disabled. Hopefully no more confusion. Thanks!

**Comment by guryonblaze** [comment]
And  I'm out. I understand the devs think this is a boon, but they also think discord is valid way to report issues.

**Reply by rcarteraz** [reply]
> This is traumatic indeed. 

Is it still doing that for you? I just tested and it's working but it could be at the time thing or maybe iOS vs Android issues?

**Reply by HTFAmIFunny** [reply]
> And I'm out. I understand the devs think this is a boon, but they also think discord is valid way to report issues.

"Out" is exactly where they were hoping most of us who ask uncomfortable questions would go.  

Hey crichton, still eating Snickers bars upside down for the texture?

**Reply by thebentern** [reply]
> 
> "Out" is exactly where they were hoping most of us who ask uncomfortable questions would go.
> 
> Hey crichton, still eating Snickers bars upside down for the texture?

I understand some may be uncomfortable with this change, but there is no conspiracy here. We did this as described: to consolidate the message board style community feedback onto the platform we utilize the most. The fact is that the Discourse was slowly dying and becoming unmoderated. This gives us better visibility and integration into ticketing. Lets refrain from being accusatory and combative.

**Reply by guryonblaze** [reply]
Is this a reply a quote wtf is this mess?

**Reply by guryonblaze** [reply]
Lol they've added a reddit bot to post about reddit threads... This is working right?

**Reply by rcarteraz** [reply]
> Lol they've added a reddit bot to post about reddit threads... This is working right?

Yes, we did. Since Discord is our largest platform, and Reddit is right behind it -- it just made sense to help expose it a little more. Neither are mandatory to use though.
